asttokens/asttokens.py

```diff
@@ -175,13 +198,15 @@ def token_range(self, first_token, last_token, include_extra=False):
         yield self._tokens[i]
 
   def get_tokens(self, node, include_extra=False):
+    # type: (AstNode, bool) -> Iterator[Token]
     """
     Yields all tokens making up the given node. If include_extra is True, includes non-coding
     tokens such as tokenize.NL and .COMMENT.
     """
-    return self.token_range(node.first_token, node.last_token, include_extra=include_extra)
+    return self.token_range(cast(Token, node.first_token), cast(Token, node.last_token), include_extra=include_extra)
```

Comment:
Doesn't `EnhancedAST` remove the need for these casts?

Reply:
Yup, removed.